server/pbenchinacan/load_keycloak.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-# This scripts loads the keycloak server with following configuration
+# This scripts creates the custom keycloak server image with following configuration
```

Comment:
This does not appear to create an image, but rather update a running KeyCloak container to have the desired Pbench realm settings / configuratino.